### ISSUE TITLE
Suggestion: make hat particles in first-person an optional toggle

### ISSUE THREAD

**Issue by PastaLaPasta** [fonnymunkey/SimpleHats#87]
Please consider adding an option in the mod's config that let's you disable particle effects from yourself if you are in first-person. Some hats, while having cool designs, are too distracting to use. I don't want to be avoiding cool hats just because they happen to be annoying to use.

**Comment by fonnymunkey** [comment]
If you are using a mod that adds first person model rendering (Which is what causes hats and particles to render in first person) set forceFirstPersonNoRender to true in the config

**Comment by PastaLaPasta** [comment]
Now that's odd: I double checked in my WIP 1.21.1 private modpack if I had particles in first-person or not, and apparently I don't despite this being a problem in older versions of the pack (though some mods aren't updated yet, so that may change). I'll just keep your tip in mind.

On the other hands, I noticed that if I opened my inventory, the particles starts rendering until I close it, but only if it's an inventory where I can see my character. Apparently the preview model in your inventory counts as being in third-person.

**Comment by fonnymunkey** [comment]
Yeah the way Minecraft handles the inventory screen is re-rendering your actual player model as though its third person, instead of like a fake version of the player, which will cause third person effects like the particles to start rendering.

**Comment by LogicEngineer** [comment]
I realize this is an old issue but it's still open. 

I'm in Neoforge 1.21.1 and I'm having issues with hat particles rendering in first person while I have the Iris Shaders mod and Complementary Shaders - Reimagined turned on. I tried setting forceFirstPersonNoRender = true in the config and that does work, but it also means the hats don't render in the inventory view which is such a shame. This seems to maybe be something that the shader pack needs to handle? 

I tested several shader packs and most seem to have this issue. I was able to find a shader pack called Mellow Shader that, curiously, does NOT have the first person particle rendering issue. Not sure why a shader pack would be enabling first person rendering. 

I'll admit I'm extremely ignorant of the workings of Minecraft's rendering processes so I have no idea what can or should be done about this. Having to choose between my favorite shader pack or annoying particles all over my screen while wearing certain hats is kind of a bummer though.
